keep/cli/cli.py

```diff
@@ -1,18 +1,31 @@
 import logging
 import logging.config
+import os
 import sys
+import uuid
 from dataclasses import fields
 from importlib import metadata
 
 import click
 import yaml
 from dotenv import find_dotenv, load_dotenv
+from posthog import Posthog
 
 from keep.alertmanager.alertmanager import AlertManager
 from keep.cli.click_extensions import NotRequiredIf
 from keep.providers.providers_factory import ProvidersFactory
 
 load_dotenv(find_dotenv())
+if not os.getenv("DISABLE_POSTHOG"):
+    posthog_api_key = (
+        os.getenv("POSTHOG_API_KEY")
+        or "phc_muk9qE3TfZsX3SZ9XxX52kCGJBclrjhkP9JxAQcm1PZ"
+    )
+    posthog_client = Posthog(api_key=posthog_api_key, host="https://app.posthog.com")
+else:
+    posthog_client = None
```

Comment:
perhaps put this in a helper and use it also in the API? currently you can't "shut" posthog for the api and this code is duplicate